### ISSUE TITLE
All values in menu, gameplay, and resultsScreen sections are stuck on default values

### ISSUE THREAD

**Issue by clxxiii** [l3lackShark/gosumemory#128]
Here's some data pasted from http://127.0.0.1:24050/json while running the client in tourney mode:

```json
{
   "settings":{
      "showInterface":true,
      "folders":{
         "game":"D:\\Users\\clxxiii\\Documents\\osu!tourney",
         "skin":"User",
         "songs":"D:\\Users\\clxxiii\\Documents\\osu!tourney\\Songs"
      }
   },
   "menu":{
      "mainMenu":{
         "bassDensity":2.614514622092247
      },
      "state":22,
      "gameMode":0,
      "isChatEnabled":0,
      "bm":{
         "time":{
            "firstObj":0,
            "current":157038,
            "full":0,
            "mp3":0
         },
         "id":0,
         "set":0,
         "md5":"",
         "rankedStatus":0,
         "metadata":{
            "artist":"",
            "artistOriginal":"",
            "title":"",
            "titleOriginal":"",
            "mapper":"",
            "difficulty":""
         },
         "stats":{
            "AR":0,
            "CS":0,
            "OD":0,
            "HP":0,
            "SR":0,
            "BPM":{
               "min":0,
               "max":0
            },
            "maxCombo":0,
            "fullSR":0,
            "memoryAR":0,
            "memoryCS":0,
            "memoryOD":0,
            "memoryHP":0
         },
         "path":{
            "full":"",
            "folder":"",
            "file":"",
            "bg":"",
            "audio":""
         }
      },
      "mods":{
         "num":0,
         "str":"NM"
      },
      "pp":{
         "100":0,
         "99":0,
         "98":0,
         "97":0,
         "96":0,
         "95":0,
         "strains":null
      }
   },
   "gameplay":{
      "gameMode":0,
      "name":"",
      "score":0,
      "accuracy":0,
      "combo":{
         "current":0,
         "max":0
      },
      "hp":{
         "normal":0,
         "smooth":0
      },
      "hits":{
         "300":0,
         "geki":0,
         "100":0,
         "katu":0,
         "50":0,
         "0":0,
         "sliderBreaks":0,
         "grade":{
            "current":"",
            "maxThisPlay":""
         },
         "unstableRate":0,
         "hitErrorArray":null
      },
      "pp":{
         "current":0,
         "fc":0,
         "maxThisPlay":0
      },
      "keyOverlay":{
         "k1":{
            "isPressed":false,
            "count":0
         },
         "k2":{
            "isPressed":false,
            "count":0
         },
         "m1":{
            "isPressed":false,
            "count":0
         },
         "m2":{
            "isPressed":false,
            "count":0
         }
      },
      "leaderboard":{
         "hasLeaderboard":false,
         "isVisible":false,
         "ourplayer":{
            "name":"",
            "score":0,
            "combo":0,
            "maxCombo":0,
            "mods":"",
            "h300":0,
            "h100":0,
            "h50":0,
            "h0":0,
            "team":0,
            "position":0,
            "isPassing":0
         },
         "slots":null
      }
   },
   "resultsScreen":{
      "name":"",
      "score":0,
      "maxCombo":0,
      "mods":{
         "num":0,
         "str":""
      },
      "300":0,
      "geki":0,
      "100":0,
      "katu":0,
      "50":0,
      "0":0
   },
   "tourney":{
      "manager":{
         "ipcState":4,
         "bestOF":1,
         "teamName":{
            "left":"",
            "right":""
         },
         "stars":{
            "left":1,
            "right":0
         },
         "bools":{
            "scoreVisible":true,
            "starsVisible":true
         },
         "chat":null,
         "gameplay":{
            "score":{
               "left":1187539,
               "right":1000964
            }
         }
      },
      "ipcClients":[
         {
            "team":"left",
            "spectating":{
               "name":"MintyDude",
               "country":"United Kingdom",
               "userID":11554464,
               "accuracy":98.14,
               "rankedScore":32791370360,
               "playCount":106043,
               "globalRank":952,
               "totalPP":10891
            },
            "gameplay":{
               "gameMode":0,
               "score":933228,
               "name":"",
               "accuracy":0,
               "hits":{
                  "300":0,
                  "geki":0,
                  "100":0,
                  "katu":0,
                  "50":0,
                  "0":0,
                  "sliderBreaks":0,
                  "unstableRate":0
               },
               "combo":{
                  "current":0,
                  "max":0
               },
               "mods":{
                  "num":0,
                  "str":""
               },
               "hp":{
                  "normal":0,
                  "smooth":0
               }
            }
         },
         {
            "team":"left",
            "spectating":{
               "name":"Noomy",
               "country":"Belgium",
               "userID":17012407,
               "accuracy":98.14,
               "rankedScore":10861122218,
               "playCount":28343,
               "globalRank":65972,
               "totalPP":4633
            },
            "gameplay":{
               "gameMode":0,
               "score":254311,
               "name":"",
               "accuracy":0,
               "hits":{
                  "300":0,
                  "geki":0,
                  "100":0,
                  "katu":0,
                  "50":0,
                  "0":0,
                  "sliderBreaks":0,
                  "unstableRate":0
               },
               "combo":{
                  "current":0,
                  "max":0
               },
               "mods":{
                  "num":0,
                  "str":""
               },
               "hp":{
                  "normal":0,
                  "smooth":0
               }
            }
         },
         {
            "team":"right",
            "spectating":{
               "name":"WhyYouSuckAtosu",
               "country":"United Kingdom",
               "userID":19564769,
               "accuracy":98.8,
               "rankedScore":5098705587,
               "playCount":38792,
               "globalRank":26780,
               "totalPP":5976
            },
            "gameplay":{
               "gameMode":0,
               "score":267351,
               "name":"",
               "accuracy":0,
               "hits":{
                  "300":0,
                  "geki":0,
                  "100":0,
                  "katu":0,
                  "50":0,
                  "0":0,
                  "sliderBreaks":0,
                  "unstableRate":0
               },
               "combo":{
                  "current":0,
                  "max":0
               },
               "mods":{
                  "num":0,
                  "str":""
               },
               "hp":{
                  "normal":0,
                  "smooth":0
               }
            }
         },
         {
            "team":"right",
            "spectating":{
               "name":"ssubinism",
               "country":"Lithuania",
               "userID":16329318,
               "accuracy":98.1,
               "rankedScore":44096550975,
               "playCount":86046,
               "globalRank":6213,
               "totalPP":8078
            },
            "gameplay":{
               "gameMode":0,
               "score":733613,
               "name":"",
               "accuracy":0,
               "hits":{
                  "300":0,
                  "geki":0,
                  "100":0,
                  "katu":0,
                  "50":0,
                  "0":0,
                  "sliderBreaks":0,
                  "unstableRate":0
               },
               "combo":{
                  "current":0,
                  "max":0
               },
               "mods":{
                  "num":0,
                  "str":""
               },
               "hp":{
                  "normal":0,
                  "smooth":0
               }
            }
         }
      ]
   }
}
```

The tourney tree is the only one that seems to be working at all. The gameplay stats under each tourney user is also stuck on default values

**Comment by ar1a** [comment]
I'm getting this too, output from the program is 
```
Checking Updates... (can take some time if you have bad routing to GitHub)
2022/04/06 11:26:47 Current binary is the latest version 1.3.5
2022/04/06 11:26:47 Release notes:
 Many thanks to @Wieku for porting the new formula to Go. Only gameplay is showing correct values for the time being.

* Fix tournament mode wrong individual score values
[MEMORY] Got osu!status addr...
[MEMORY] Songs folder: E:\osu songs
[MEMORY] Resolving patterns...
[MEMORY] Got all patterns...
WARNING: Mania pp calcualtion is experimental and only works if you choose mania gamemode in the SongSelect!
Initialization complete, you can now visit http://127.0.0.1:24050 or add it as a browser source in OBS
[GAMEOVERLAY] Initialized successfully, see https://github.com/l3lackShark/gosumemory/wiki/GameOverlay for tutorial
"Expected mp3, got something else. Aborting mp3 time calculation. GOT: " ""
&fmt.wrapError{
  msg: "Failed to calc via wieku calculator, falling back to oppai, ERROR: open : The system cannot find the file specified.",
  err: &fs.PathError{
    Op:   "open",
    Path: "",
    Err:  0x2,
  },
}```

**Comment by ar1a** [comment]
seems something changed in cutting edge. simple fix for now is back to stable

**Comment by clxxiii** [comment]
It doesn't seem to be working on stable either.

**Comment by l3lackShark** [comment]
Please check #announcements in our [Discord](https://discord.com/invite/8enr4qD) server for an immediate fix. Expect to get this released later today.

**Comment by l3lackShark** [comment]
Should be fixed in 1.3.6
